cargo-near-build/src/near/build/mod.rs

```diff
@@ -234,6 +234,8 @@ fn is_newer_than(prev: &Utf8PathBuf, next: &Utf8PathBuf) -> bool {
     prev_time > next_time
 }
```

Comment:
// todo!: merge pr with rust_toolchain pin onto this pr, add a WARNING with 15sec sleep about 1.87 toolchain not digestible yet, despite being builable